src/types/formatted/healthCareLocalFacility.ts

```diff
@@ -10,4 +16,10 @@ export type HealthCareLocalFacility = PublishedEntity & {
   path: string
   administration?: { entityId: number }
   vamcEhrSystem: VamcEhr['field_region_page']['field_vamc_ehr_system']
+  officeHours: FieldOfficeHours[]
+  address: FieldAddress
+  phoneNumber: string
+  image: MediaImage
+  facilityLocatorApiId: string
+  geoLocation: FieldGeoLocation
```

Comment:
It _is_ admittedly strange to "translate" some of these properties into a formatted structure but not translate others. Buuuuuuut... 🤷 Formatting everything is a slog and doesn't actually give us any value.